students/rick_halsell/lesson04/mailroom_part2.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
+import os
+import sys
+import operator
+from operator import itemgetter
```

Comment:
Forgot to import time?

Reply:
Yes, I added it back to my updated version.